src/main/scala/com/microsoft/hyperspace/index/execution/BucketUnionExec.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index.execution
+
+import scala.reflect.ClassTag
+
+import org.apache.spark.{OneToOneDependency, Partition, SparkContext, TaskContext}
+import org.apache.spark.rdd.PartitionerAwareUnionRDD
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.catalog.BucketSpec
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.catalyst.plans.physical.Partitioning
+import org.apache.spark.sql.execution.SparkPlan
+
+import com.microsoft.hyperspace.index.plans.logical.BucketUnion
+
+// scalastyle:off filelinelengthchecker
```

Comment:
Do you want to keep disable this style rule for the final merge?

Reply:
Yes it's because of the below URL. Please let me know if there's any other way..

Reply:
@sezruby Can you use the direct link to the video? https://youtu.be/7cvaH33S7uc

Reply:
Thanks! I replaced with the url & removed scalastyle:off